packages/features/timezone-buddy/components/AvailabilityEditSheet.tsx

```diff
@@ -119,6 +121,7 @@ export function AvailabilityEditSheetForm(props: Props & { data: Data; isPending
   const updateMutation = trpc.viewer.availability.schedule.update.useMutation({
     onSuccess: async () => {
       await utils.viewer.availability.listTeam.invalidate();
+      await utils.viewer.availability.schedule.getAllSchedulesByUserId.invalidate({ userId });
```

Comment:
Invalidate cache or else it would show stale data on closing and opening the sheet